src/App.js

```diff
@@ -723,6 +724,22 @@ function App() {
   const [data, setData] = useState<ReactProfilerData | null>(null);
   const [flamechart, setFlamechart] = useState<FlamechartData | null>(null);
   const [schedulerCanvasHeight, setSchedulerCanvasHeight] = useState<number>(0);
+  const [JSON_PATH, setJSON_PATH] = useState('../static/small-devtools.json'); //TODO: add type
+  
+  const inputProfilerData = useCallback(
+    (file) => {
+
+      let fileName = file.target.files[0];
+
+      const regex = /^.*\.json$/g;
+      
+      if(fileName.name.match(regex)){
+        console.log(JSON_PATH);
+        setJSON_PATH = file;
```

Comment:
`setJSON_PATH` is a function

```suggestion
        setJSON_PATH(fileName);
```